doc/nrf/libraries/caf/leds.rst

```diff
@@ -224,13 +224,13 @@ The following code snippets show examples of the DTS nodes:
 
 		pwm_led3: led_pwm_3 {
 			status = "okay";
-			pwms = <&pwm1 4>;
+			pwms = <&pwm1 0 PWM_MSEC(20) PWM_POLARITY_NORMAL>;
 			label = "LED1";
 		};
 	};
 
      In this example, ``pwmleds0`` is a tri-channel color LED node, while ``pwmleds1`` is a monochromatic LED node.
-     Both ``pwmleds`` nodes are pointing to the ``pwms`` properties corresponding to PWM nodes set in Example 2 in `Enabling the PWM ports`_, with the respective channel GPIO pin numbers.
+     Both ``pwmleds`` nodes are pointing to the ``pwms`` properties corresponding to PWM nodes set in Example 2 in `Enabling the PWM ports`_, with the respective channel numbers.
```

Comment:
These two sentences are appearing inside the code block. Is it supposed to be like that? (See the 'Example 2' section in http://developer.nordicsemi.com/nRF_Connect_SDK_dev/doc/PR-7597/nrf/libraries/caf/leds.html#enabling-the-led-pwm-nodes)

Reply:
Right. I corrected the indentation.